cnd/src/storage/db/tables.rs

```diff
@@ -500,6 +447,47 @@ impl BtcDaiOrder {
 
         Ok(params)
     }
+
+    /// Move the amount that is settling from open to settling.
+    ///
+    /// Whilst we don't have partial order matching, this simply means updating
+    /// `settling` to the amount of `open` and updating `open` to `0`.
+    ///
+    /// Once we implement partial order matching, this will need to get more
+    /// sophisticated.
+    pub fn mark_as_settling(&self, conn: &SqliteConnection) -> Result<()> {
```

Comment:
```suggestion
    pub fn set_to_settling(&self, conn: &SqliteConnection) -> Result<()> {
```
`mark` is not a terminology usually used. `set` is more common to Rust.